### ISSUE TITLE
UI do not show

### ISSUE THREAD

**Issue by kolyshkin** [kgshank/gse-sound-output-device-chooser#107]
Please provide the shell and extension information. It will be easier for me to understand the issue without too many unnecessary questions

**Describe the bug**

There are no controls/UI shown in the menu, like the extension is not installed. I have tried to remove and reinstall the extension, reboot the system etc -- all to no avail.

There is nothing suspicious in the logs (see below), and I don't have a slightest idea about how to debug it :( 

Extension settings are shown (in e.g. gnome-tweaks or when you select settings on the , but its UI is gone.

**Extension Information**
* Shell Version
```
$ rpm -q gnome-shell
gnome-shell-3.36.4-1.fc32.x86_64
$ gnome-shell --version
GNOME Shell 3.36.4
```
* Extension Version
31

* Install method (extension.gnome.org, AUR etc.)
https://extensions.gnome.org/extension/906/sound-output-device-chooser/

* OS
Fedora 32 with latest update as of today (Mon Jul 20)

**Error / Journal messages**
* Journal messages related to the error / extension.

Jul 20 12:46:08 kir-rhat gnome-shell[5885]: Archive:  /tmp/OGD7N0.shell-extension.zip
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/icons/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/de_DE/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/de_DE/LC_MESSAGES/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/it_IT/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/it_IT/LC_MESSAGES/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/pt_BR/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/pt_BR/LC_MESSAGES/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/pt_PT/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/pt_PT/LC_MESSAGES/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/sk/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/sk/LC_MESSAGES/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/schemas/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/ui/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:    creating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/utils/
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/base.js
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/convenience.js
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/extension.js
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/icons/blank.png
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/license
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/de_DE/LC_MESSAGES/sound-output-device-chooser.po
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/it_IT/LC_MESSAGES/sound-output-device-chooser.po
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/pt_BR/LC_MESSAGES/sound-output-device-chooser.po
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/pt_PT/LC_MESSAGES/sound-output-device-chooser.po
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/locale/sk/LC_MESSAGES/sound-output-device-chooser.po
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/prefs.js
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/schemas/gschemas.compiled
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/schemas/org.gnome.shell.extensions.sound-output-device-chooser.gschema.xml
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/sound-output-device-chooser.pot
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/ui/prefs-dialog.gtkbuilder
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/utils/libpulse_introspect.py
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:   inflating: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/utils/pa_helper.py
Jul 20 12:46:08 kir-rhat gnome-shell[5885]:  extracting: /home/kir/.local/share/gnome-shell/extensions/sound-output-device-chooser@kgshank.net/metadata.json
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: SDC: init
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Constructoroutput
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Max Id:32
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards iec958-stereo-output for card id 0
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards iec958-stereo-output for card id 1
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Digital Output (S/PDIF) (USB Audio)
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:iec958-stereo
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Added: 3:Digital Output (S/PDIF):iec958-stereo-output
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile form stream id analog-output-headphones
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Headphones (USB Audio)
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:analog-stereo
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Added: 4:Headphones:analog-output-headphones
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Activated: 4
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-4 for card id 0
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-4 for card id 1
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-4 for card id 2
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:HDMI / DisplayPort 5 (Built-in Audio)
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-stereo-extra4
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround-extra4
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround71-extra4
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Added: 7:HDMI / DisplayPort 5:hdmi-output-4
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: Removed: 7
Jul 20 12:46:08 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-3 for card id 0
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-3 for card id 1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-3 for card id 2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:HDMI / DisplayPort 4 (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-stereo-extra3
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround-extra3
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround71-extra3
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 8:HDMI / DisplayPort 4:hdmi-output-3
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Removed: 8
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-2 for card id 0
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-2 for card id 1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-2 for card id 2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:HDMI / DisplayPort 3 (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-stereo-extra2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround-extra2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround71-extra2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 9:HDMI / DisplayPort 3:hdmi-output-2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Removed: 9
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-1 for card id 0
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-1 for card id 1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-1 for card id 2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:HDMI / DisplayPort 2 (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-stereo-extra1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround-extra1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-surround71-extra1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 10:HDMI / DisplayPort 2:hdmi-output-1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Removed: 10
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-0 for card id 0
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-0 for card id 1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards hdmi-output-0 for card id 2
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:HDMI / DisplayPort (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:hdmi-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 11:HDMI / DisplayPort:hdmi-output-0
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile form stream id analog-output-headphones
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Headphones (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:analog-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 12:Headphones:analog-output-headphones
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile form stream id analog-output-speaker
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Speakers (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:output:analog-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 13:Speakers:analog-output-speaker
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Removed: 13
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: SDC: init
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Constructorinput
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Max Id:33
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile form stream id iec958-stereo-input
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Digital Input (S/PDIF) (HD Pro Webcam C920)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:input:iec958-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 1:Digital Input (S/PDIF):iec958-stereo-input
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Activated: 1
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards analog-input-mic for card id 0
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Microphone (HD Pro Webcam C920)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:input:analog-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 2:Microphone:analog-input-mic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: New logic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile from cards iec958-stereo-input for card id 0
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Digital Input (S/PDIF) (USB Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:input:iec958-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 5:Digital Input (S/PDIF):iec958-stereo-input
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile form stream id analog-input-mic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Microphone (USB Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:input:analog-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 6:Microphone:analog-input-mic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile form stream id analog-input-mic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Microphone (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:input:analog-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 14:Microphone:analog-input-mic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Removed: 14
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Getting profile form stream id analog-input-internal-mic
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Device Name:Internal Microphone (Built-in Audio)
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Profile:input:analog-stereo
Jul 20 12:46:09 kir-rhat gnome-shell[1873]: SDC Debug: Added: 15:Internal Microphone:analog-input-internal-mic

**Comment by kolyshkin** [comment]
At the same time, pavucontrol works, all other gnome extensions work.

Anything I can do to debug it?

**Comment by kgshank** [comment]
Do you have any other extension that modifies the volume menu or status menu?

**Comment by kolyshkin** [comment]
Just found the reason. I use dual monitor setup (a laptop plus an external monitor), and the top bar is supposed to be the same on both, and yet I just found it is _slightly_ different. This might be because of gnome 3.36.4 upgrade (or not).

Does not matter which monitor do I choose as primary -- now the device chooser is only shown on a laptop monitor, and is not shown on an external monitor.

I found out that this is what dash to panel extension does. I guess this issue can be closed, sorry for wasting your time.
